tests/e2e_testdata.go

```diff
@@ -0,0 +1,514 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/adshmh/meter/db"
+	"github.com/gojektech/heimdall/httpclient"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	influxAPI "github.com/influxdata/influxdb-client-go/v2/api"
+	influxDomain "github.com/influxdata/influxdb-client-go/v2/domain"
+	"github.com/pokt-foundation/portal-api-go/repository"
+	stringUtils "github.com/pokt-foundation/utils-go/strings"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO move this to utils-go
+func startOfDay(day time.Time) time.Time {
+	y, m, d := day.Date()
+	l := day.Location()
+
+	return time.Date(y, m, d, 0, 0, 0, 0, l)
+}
+
+var (
+	// Test vars
+	ctx               = context.Background()
+	today             = startOfDay(time.Now().UTC())
+	endOfDay          = today.AddDate(0, 0, 1)
```

Comment:
Technically this goes over to the start of the next day, is this the intended value (considering the name is `endOfDay`)?

Reply:
Yes you are right. It goes to exactly midnight of the next day which is arguably the "end of the same day" but I can just subtract a ms from it to be exactly correct.